packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -44,6 +46,10 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
     const indexOfSelection = selected_country =>
         residence_list.findIndex(item => item.text.toLowerCase() === selected_country.toLowerCase());
 
+    const checkBoxRequired = selected_country => [localize('Brazil')].includes(selected_country);
```

Comment:
```suggestion
    const checkBoxRequired = selected_country => [localize('Brazil')].includes(selected_country);
```
using localize output as part of a logic check is always a bad idea. is there any other unique identifiers for brazil u can use?